### ISSUE TITLE
Error Associated with chromedriver-binary-auto When Installing Using pip install blobtoolkit

### ISSUE THREAD

**Issue by foreignsand** [blobtoolkit/blobtoolkit#214]
I'm trying to troubleshoot this but cannot figure out what the problem might be. Install seems to be choking on `chromedriver-binary-auto`, but then this is already installed on my machine. The error is quite long. The first salient bit occurs after everything has gone smoothly and pip attempts to build a wheel for `chromedriver-binary-auto`:

```
Building wheels for collected packages: chromedriver-binary-auto
  Building wheel for chromedriver-binary-auto (setup.py) ... error
  ERROR: Command errored out with exit status 1:
```

The next bit is here, when pip moves on to attempt to install `chromedriver-binary-auto` using setup.py:

```
 ERROR: Failed building wheel for chromedriver-binary-auto
  Running setup.py clean for chromedriver-binary-auto
Failed to build chromedriver-binary-auto
Installing collected packages: chromedriver-binary-auto, blobtoolkit
  Attempting uninstall: chromedriver-binary-auto
    Found existing installation: chromedriver-binary-auto 0.3.1
    Uninstalling chromedriver-binary-auto-0.3.1:
      Successfully uninstalled chromedriver-binary-auto-0.3.1
    Running setup.py install for chromedriver-binary-auto ... error
    ERROR: Command errored out with exit status 1:
```

And finally, when it rolls back the uninstall of `chromedriver-binary-auto`:

```
    RuntimeError: Failed to find release information: https://chromedriver.storage.googleapis.com/LATEST_RELEASE_124
    ----------------------------------------
  Rolling back uninstall of chromedriver-binary-auto
  Moving to /Users/lemurbear/opt/anaconda3/bin/chromedriver-path
   from /private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-uninstall-_cbcg3d1/chromedriver-path
  Moving to /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/chromedriver_binary/
   from /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/~hromedriver_binary
  Moving to /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/chromedriver_binary_auto-0.3.1.dist-info/
   from /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/~hromedriver_binary_auto-0.3.1.dist-info
ERROR: Command errored out with exit status 1
```

If I build a wheel for `chromedriver-binary-auto` using `pip`, there are no issues because the package is installed and is already a wheel:

```
(base) lemurbear@MacBook-Pro-10 blobtoolkit % pip wheel chromedriver-binary-auto
Processing /Users/lemurbear/Library/Caches/pip/wheels/1f/2d/78/1b53f8f3caa66a6418f96e1d510d7f2166bb2342ebd51ee65e/chromedriver_binary_auto-0.3.1-py3-none-any.whl
  File was already downloaded /Users/lemurbear/blobtoolkit/chromedriver_binary_auto-0.3.1-py3-none-any.whl
Skipping chromedriver-binary-auto, due to already being wheel.
```

I'm not sure what I need to do to move this forward. Any help would be much appreciated!

Cheers,
Emily

Here is the entire block of output from the attempted install:

```
(base) lemurbear@MacBook-Pro-10 blobtoolkit % pip install blobtoolkit           
Processing /Users/lemurbear/Library/Caches/pip/wheels/89/72/e2/35715c941787c24c4d0995b5d9eac5d718b44e549b9c4f04f6/blobtoolkit-4.1.5-py3-none-any.whl
Requirement already satisfied: tolkein>=0.5.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (0.5.0)
Requirement already satisfied: ujson>=5.7.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (5.9.0)
Requirement already satisfied: pyvirtualdisplay==3.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (3.0)
Requirement already satisfied: docopt>=0.6.2 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (0.6.2)
Requirement already satisfied: geckodriver-autoinstaller==0.1.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (0.1.0)
Collecting chromedriver-binary-auto==0.2.3
  Using cached chromedriver-binary-auto-0.2.3.tar.gz (5.1 kB)
Requirement already satisfied: selenium==4.7.2 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (4.7.2)
Requirement already satisfied: pyyaml in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (5.3.1)
Requirement already satisfied: fastjsonschema==2.15.3 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (2.15.3)
Requirement already satisfied: tqdm==4.64.1 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (4.64.1)
Requirement already satisfied: blobtk>=0.2.3 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (0.5.3)
Requirement already satisfied: psutil==5.9.4 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from blobtoolkit) (5.9.4)
Requirement already satisfied: xmltodict>=0.12.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from tolkein>=0.5.0->blobtoolkit) (0.13.0)
Requirement already satisfied: requests>=2.24.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from tolkein>=0.5.0->blobtoolkit) (2.31.0)
Requirement already satisfied: urllib3[socks]~=1.26 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from selenium==4.7.2->blobtoolkit) (1.26.18)
Requirement already satisfied: trio-websocket~=0.9 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from selenium==4.7.2->blobtoolkit) (0.11.1)
Requirement already satisfied: trio~=0.17 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from selenium==4.7.2->blobtoolkit) (0.25.0)
Requirement already satisfied: certifi>=2021.10.8 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from selenium==4.7.2->blobtoolkit) (2023.7.22)
Requirement already satisfied: idna<4,>=2.5 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from requests>=2.24.0->tolkein>=0.5.0->blobtoolkit) (2.10)
Requirement already satisfied: charset-normalizer<4,>=2 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from requests>=2.24.0->tolkein>=0.5.0->blobtoolkit) (3.3.2)
Requirement already satisfied: PySocks!=1.5.7,<2.0,>=1.5.6; extra == "socks" in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from urllib3[socks]~=1.26->selenium==4.7.2->blobtoolkit) (1.7.1)
Requirement already satisfied: exceptiongroup; python_version < "3.11" in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (1.2.1)
Requirement already satisfied: wsproto>=0.14 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (1.2.0)
Requirement already satisfied: outcome in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.3.0.post0)
Requirement already satisfied: attrs>=23.2.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (23.2.0)
Requirement already satisfied: sortedcontainers in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (2.2.2)
Requirement already satisfied: sniffio>=1.3.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from trio~=0.17->selenium==4.7.2->blobtoolkit) (1.3.1)
Requirement already satisfied: h11<1,>=0.9.0 in /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages (from wsproto>=0.14->trio-websocket~=0.9->selenium==4.7.2->blobtoolkit) (0.14.0)
Building wheels for collected packages: chromedriver-binary-auto
  Building wheel for chromedriver-binary-auto (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/lemurbear/opt/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py'"'"'; __file__='"'"'/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-wheel-h1lz3per
       cwd: /private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/
  Complete output (59 lines):
  running bdist_wheel
  running build
  running build_py
  Traceback (most recent call last):
    File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/chromedriver_binary/utils.py", line 95, in get_latest_release_for_version
      response = urlopen(release_url, context=ssl.SSLContext(protocol=ssl.PROTOCOL_TLS))
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 222, in urlopen
      return opener.open(url, data, timeout)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 531, in open
      response = meth(req, response)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 640, in http_response
      response = self.parent.error(
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 569, in error
      return self._call_chain(*args)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 502, in _call_chain
      result = func(*args)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 649, in http_error_default
      raise HTTPError(req.full_url, code, msg, hdrs, fp)
  urllib.error.HTTPError: HTTP Error 404: Not Found
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py", line 62, in <module>
      setup(
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/__init__.py", line 103, in setup
      return distutils.core.setup(**attrs)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 185, in setup
      return run_commands(dist)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
      dist.run_commands()
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
      self.run_command(cmd)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/dist.py", line 963, in run_command
      super().run_command(command)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
      cmd_obj.run()
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/wheel/bdist_wheel.py", line 223, in run
      self.run_command('build')
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
      self.distribution.run_command(command)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/dist.py", line 963, in run_command
      super().run_command(command)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
      cmd_obj.run()
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/command/build.py", line 131, in run
      self.run_command(cmd_name)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
      self.distribution.run_command(command)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/dist.py", line 963, in run_command
      super().run_command(command)
    File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
      cmd_obj.run()
    File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py", line 31, in run
      chromedriver_version = get_latest_release_for_version(chrome_major)
    File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/chromedriver_binary/utils.py", line 100, in get_latest_release_for_version
      raise RuntimeError('Failed to find release information: {}'.format(release_url))
  RuntimeError: Failed to find release information: https://chromedriver.storage.googleapis.com/LATEST_RELEASE_124
  ----------------------------------------
  ERROR: Failed building wheel for chromedriver-binary-auto
  Running setup.py clean for chromedriver-binary-auto
Failed to build chromedriver-binary-auto
Installing collected packages: chromedriver-binary-auto, blobtoolkit
  Attempting uninstall: chromedriver-binary-auto
    Found existing installation: chromedriver-binary-auto 0.3.1
    Uninstalling chromedriver-binary-auto-0.3.1:
      Successfully uninstalled chromedriver-binary-auto-0.3.1
    Running setup.py install for chromedriver-binary-auto ... error
    ERROR: Command errored out with exit status 1:
     command: /Users/lemurbear/opt/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py'"'"'; __file__='"'"'/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-record-5xoduwld/install-record.txt --single-version-externally-managed --compile --install-headers /Users/lemurbear/opt/anaconda3/include/python3.8/chromedriver-binary-auto
         cwd: /private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/
    Complete output (74 lines):
    running install
    /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/cmd.py:66: SetuptoolsDeprecationWarning: setup.py install is deprecated.
    !!
    
            ********************************************************************************
            Please avoid running ``setup.py`` directly.
            Instead, use pypa/build, pypa/installer or other
            standards-based tools.
    
            See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
            ********************************************************************************
    
    !!
      self.initialize_options()
    running build
    running build_py
    Traceback (most recent call last):
      File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/chromedriver_binary/utils.py", line 95, in get_latest_release_for_version
        response = urlopen(release_url, context=ssl.SSLContext(protocol=ssl.PROTOCOL_TLS))
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 222, in urlopen
        return opener.open(url, data, timeout)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 531, in open
        response = meth(req, response)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 640, in http_response
        response = self.parent.error(
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 569, in error
        return self._call_chain(*args)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 502, in _call_chain
        result = func(*args)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/urllib/request.py", line 649, in http_error_default
        raise HTTPError(req.full_url, code, msg, hdrs, fp)
    urllib.error.HTTPError: HTTP Error 404: Not Found
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py", line 62, in <module>
        setup(
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/__init__.py", line 103, in setup
        return distutils.core.setup(**attrs)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 185, in setup
        return run_commands(dist)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
        dist.run_commands()
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
        self.run_command(cmd)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/dist.py", line 963, in run_command
        super().run_command(command)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
        cmd_obj.run()
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/command/install.py", line 78, in run
        return orig.install.run(self)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/command/install.py", line 697, in run
        self.run_command('build')
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
        self.distribution.run_command(command)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/dist.py", line 963, in run_command
        super().run_command(command)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
        cmd_obj.run()
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/command/build.py", line 131, in run
        self.run_command(cmd_name)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/cmd.py", line 318, in run_command
        self.distribution.run_command(command)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/dist.py", line 963, in run_command
        super().run_command(command)
      File "/Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
        cmd_obj.run()
      File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py", line 31, in run
        chromedriver_version = get_latest_release_for_version(chrome_major)
      File "/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/chromedriver_binary/utils.py", line 100, in get_latest_release_for_version
        raise RuntimeError('Failed to find release information: {}'.format(release_url))
    RuntimeError: Failed to find release information: https://chromedriver.storage.googleapis.com/LATEST_RELEASE_124
    ----------------------------------------
  Rolling back uninstall of chromedriver-binary-auto
  Moving to /Users/lemurbear/opt/anaconda3/bin/chromedriver-path
   from /private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-uninstall-_cbcg3d1/chromedriver-path
  Moving to /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/chromedriver_binary/
   from /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/~hromedriver_binary
  Moving to /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/chromedriver_binary_auto-0.3.1.dist-info/
   from /Users/lemurbear/opt/anaconda3/lib/python3.8/site-packages/~hromedriver_binary_auto-0.3.1.dist-info
ERROR: Command errored out with exit status 1: /Users/lemurbear/opt/anaconda3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py'"'"'; __file__='"'"'/private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-install-yaz9c2v5/chromedriver-binary-auto/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/4n/0ghjrzsd4g393zjlgxy9ympc0000gn/T/pip-record-5xoduwld/install-record.txt --single-version-externally-managed --compile --install-headers /Users/lemurbear/opt/anaconda3/include/python3.8/chromedriver-binary-auto Check the logs for full command output.
```

**Comment by rjchallis** [comment]
I think this may be due to the blobtoolkit packages being out of date for the python version in your base conda environment. It's usually safer to install tools into separate environments so they can have their own set of dependencies, and for blobtoolkit you can use a newer python version, so something like
```
conda create -n btk_env python=3.12
conda activate btk-env
pip install blobtoolkit
``` 
The newer versions (available for python 3.9 up) also split off some of the extra dependency installs so for basic use it doesn't even need to load the chromedriver dependency.

**Comment by foreignsand** [comment]
thank you! i feel a bit guilty [and idiotic] about opening this issue because it turns out i was looking at a deprecated blobtoolkit branch for installation instructions. after looking at the actual [BlobToolKit](https://blobtoolkit.genomehubs.org) site, i quickly and successfully installed [blobtoolkit](https://blobtoolkit.genomehubs.org/pipeline/pipeline-tutorials/configuring-the-pipeline/) in a conda environment using the following command:

```
conda create -y -n btk -c conda-forge python=3.9
conda activate btk
pip install "blobtoolkit[full]"
```

now i just have to wrap my head around how to use blobtoolkit to decontaminate my _de novo_ assembly...

thanks again!

cheers,
emily
